lib/api/rule.go

```diff
@@ -115,9 +120,15 @@ type EntityRule struct {
 	// NotTag is the negated version of the Tag field.
 	NotTag string `json:"notTag,omitempty" validate:"omitempty,tag"`
 
-	// NotNet is the negated version of the Net field.
+	// NotNet is an optional field that restricts the rule to only apply to traffic that
+	// does not originate from (or terminate at) an IP address in the given subnet.
+	// Deprecated: superseded by NotNets.
```

Comment:
Spurred by the conversation in slack - is there a good place to add a warning log when users use `Nets`, so we can indicate that it's deprecated?